examples/cdi-events/README_pt.adoc

```diff
@@ -0,0 +1,33 @@
+= CDI Events - Loose Coupling and Extensibility
+:index-group: CDI
+:jbake-type: page
+:jbake-status: published
+
+O CDI permite estender o código de negócios pelo padrão Notificador / Observador.
+
+Para fazer isso, basta injetar um campo `Event` na classe notifier. Essa classe é um modelo e o tipo de parâmetro é o tipo de objeto a ser disparado. Então, quando você quiser notificar observadores, você simplesmente chamará o método fire do objeto de evento passando como parâmetro o próprio evento.
+(sua própria classe!)
+
+On the other side you annotated a method parameter `@Observes` and the parameter type is the sent type
```

Comment:
Hi Daniel! I think at this point you forgot to translate =)

Reply:
thanks.